examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -33,6 +38,9 @@
 epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
                     proj=True, baseline=(None, 0), preload=True,
                     reject=dict(grad=4000e-13, eog=150e-6))
+# Pull 2**n points to speed up computation
+epochs = EpochsArray(epochs.get_data()[..., :1024], epochs.info,
+                     epochs.events, tmin, epochs.event_id)
```

Comment:
How long ago did you have that problem? We changed to using something that is a little bit more lenient with the floating point arithmetic that goes on under the hood about a year ago IIRC.